examples/root_robots/cliff_sensor.py

```diff
@@ -0,0 +1,14 @@
+#
+# Licensed under 3-Clause BSD license available in the License file. Copyright (c) 2021-2022 iRobot Corporation. All rights reserved.
+#
+
+from irobot_edu_sdk.backend.bluetooth import Bluetooth
+from irobot_edu_sdk.robots import event, hand_over, Color, Robot, Root
```

Comment:
no need to include `Color`

Reply:
What about `hand_over` and `Robot` both are note used as well.

Reply:
That's true. I'd say you are welcome to remove those, as well.